### ISSUE TITLE
learning_rate – it can be a function of the current progress (from 1 to 0)

### ISSUE THREAD

**Issue by drose188** [hill-a/stable-baselines#791]
Is there an example code how to make the the learning rate as a function adaptive to the current progress? 
Can gamma be an adaptive function also?

**Comment by araffin** [comment]
>Is there an example code how to make the the learning rate as a function adaptive to the current progress?

In the rl zoo, yes.
Duplicate of https://github.com/hill-a/stable-baselines/issues/509

> Can gamma be an adaptive function also?

For now, no.
